source/modules/MEI.analytical.xml

```diff
@@ -242,17 +242,28 @@
   </classSpec>
   <classSpec ident="att.keySig.anl" module="MEI.analytical" type="atts">
     <desc xml:lang="en">Analytical domain attributes.</desc>
-    <classes>
-      <memberOf key="att.accidental"/>
-      <memberOf key="att.pitch"/>
-    </classes>
     <attList>
+      <attDef ident="accid" usage="opt">
+        <gloss versionDate="2023-06-14" xml:lang="en">accidental</gloss>
+        <desc xml:lang="en">Contains an accidental for the tonic key, if one is required, <abbr>e.g.</abbr>, if <att>pname</att>
+          equals <val>c</val> and <att>accid</att> equals <val>s</val>, then a tonic of C# is indicated.</desc>
+        <datatype>
+          <rng:ref name="data.ACCIDENTAL.GESTURAL"/>
```

Comment:
Perhaps it would make more sense to only allow `data.ACCIDENTAL.GESTURAL.basic` here, since allowing the full datatype also allows extended, aeu, and persian which I don't think is necessary?